android/res/values/dimens.xml

```diff
@@ -115,6 +115,7 @@
   <dimen name="nav_menu_height">70sp</dimen>
   <dimen name="nav_menu_content_height">64dp</dimen>
   <dimen name="nav_menu_landscape_width">360dp</dimen>
+  <dimen name="nav_lane">48dp</dimen>
```

Comment:
How does it look on a smaller mdpi Android 5 device?

Reply:
Here it is on the latest version (with height defined as a percentage).
![Organic Maps with turn lane assistance on a smaller device](https://user-images.githubusercontent.com/17197562/226088225-9ea0b581-5293-4083-8e27-d88d96837d41.png)

Reply:
Thanks! That layout definitely should be improved in the future in separate pull requests. We had some issues about it already. There is not much map viewport left visible on the screen.

Interesting, how other nav apps are looking on similar small screens?